x-pack/plugins/task_manager/server/task_claimers/strategy_mget.ts

```diff
@@ -357,7 +357,10 @@ async function searchAvailableTasks({
       // Task must be enabled
       EnabledTask,
       // a task type that's not excluded (may be removed or not)
-      OneOfTaskTypes('task.taskType', claimPartitions.unlimitedTypes),
+      OneOfTaskTypes(
+        'task.taskType',
+        claimPartitions.unlimitedTypes.concat(Array.from(removedTypes))
```

Comment:
We have a test to ensure `removedTypes` get marked as `unrecognized`, this should fix it.